apps/constants/base.py

```diff
@@ -5,20 +5,19 @@
 
 
 # Add-on and File statuses.
-STATUS_NULL = 0
-STATUS_UNREVIEWED = 1
-STATUS_PENDING = 2
-STATUS_NOMINATED = 3
-STATUS_PUBLIC = 4
-STATUS_DISABLED = 5
+STATUS_NULL = 0  # No review type chosen yet, add-on is incomplete.
+STATUS_UNREVIEWED = 1  # Waiting for prelim review.
+STATUS_PENDING = 2  # Personas (lightweight themes) waiting for review.
+STATUS_NOMINATED = 3  # Waiting for full review.
+STATUS_PUBLIC = 4  # Fully reviewed.
```

Comment:
pending update is only when an addon is reviewed (`STATUS_PUBLIC` or `STATUS_LITE`), but one of its version has unreviewed files.